internal/pkg/skuba/addons/dex.go

```diff
@@ -233,17 +233,8 @@ rules:
   resources: ["customresourcedefinitions"]
   verbs: ["create", "get", "list", "update", "watch"]
 - apiGroups: ["dex.coreos.com"]
-  resources: ["oauth2clients", "connectors", "passwords", "refreshtokens"]
-  verbs: ["get", "list"]
-- apiGroups: ["dex.coreos.com"]
-  resources: ["signingkeies"]
-  verbs: ["create", "get", "list", "update"]
-- apiGroups: ["dex.coreos.com"]
-  resources: ["authcodes", "authrequests", "offlinesessionses"]
-  verbs: ["create", "delete", "get", "list", "update"]
-- apiGroups: ["dex.coreos.com"]
-  resources: ["refreshtokens"]
-  verbs: ["create", "delete"]
+  resources: ["*"]
```

Comment:
I prefer to give the permission that is missing instead of full permission to all resources.

Reply:
We will not know what's the resources dex needed. Furthermore, when dex bump version.

For now, we add more `verbs` when someone report issue and the user experience is not good.

Reply:
Yes, the custom resources defined by upstream project, we will don't know if any side effect coming later. We need to fix this to allow all verbs for dex resource for dex service account.

Reply:
just a nit, will be great having a comment in the code explaining that this is following the upstream example and a link to it https://github.com/dexidp/dex/blob/master/examples/k8s/dex.yaml#L116-L126

Reply:
+1 to @aojea comment, but with a permalink please :) (not a link to `master): https://github.com/dexidp/dex/blob/4bede5eb80822fc3a7fc9edca0ed2605cd339d17/examples/k8s/dex.yaml#L116-L126